app/controllers/hq_controller.rb

```diff
@@ -1,4 +1,15 @@
 class HqController < ApplicationController
+  NAVIGATION_BUTTONS= [
+    { text: 'OSRA', href: 'root_path', path_regex: /^\/$/, glyph: 'glyphicon-home' },
+    { text: 'Dashboard', href: '"/404.html"', path_regex: /^\/hq(\/)?$/, glyph: 'glyphicon-dashboard' },
+    { text: 'Admin Users', href: '"/404.html"', path_regex: /^\/hq\/admin_users/, glyph: 'glyphicon-user' },
+    { text: 'Comments', href: '"/404.html"', path_regex: /^\/hq\/comments/, glyph: 'glyphicon-comment' },
+    { text: 'Orphans', href: '"/404.html"', path_regex: /^\/hq\/orphans/, glyph: 'glyphicon-user' },
+    { text: 'Partners', href: 'hq_partners_path', path_regex: /^\/hq\/partners/, glyph: 'glyphicon-user' },
+    { text: 'Sponsors', href: 'hq_sponsors_path', path_regex: /^\/hq\/sponsors/, glyph: 'glyphicon-user' },
+    { text: 'Users', href: 'hq_users_path', path_regex: /^\/hq\/users/, glyph: 'glyphicon-user' }
+  ]
```

Comment:
What is gained by this approach? The controllers typically link the views to models and/or contain certain amounts of logic that one doesn't want to pollute the view with. Neither is the case here, all that's needed is to render a handful of nav buttons. Why not do everything in the partial?